### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by p6002** [RestyaPlatform/board#4426]
Pages don't work, docker compose puts up an incomplete image I think. any alternatives? 

<img width="930" alt="image" src="https://github.com/RestyaPlatform/board/assets/74392057/e8e5a04e-c324-49a4-892f-40ad4bc71ec6">

**Comment by wilsonwu** [comment]
Seems Yes

**Comment by KaKi87** [comment]
> any alternatives?

I'm wondering as well

**Comment by chmac** [comment]
kanboard, wekan, and taiga are the most popular and actively maintained alternatives I have found.

**Comment by KaKi87** [comment]
Kanboard's UX leaves much to be desired, Wekan is complicated to self-host, and Taiga is too much when you only want a kanban. :sweat_smile:

**Comment by chmac** [comment]
I agree with all of those. Then I later yesterday found planka, and it seems like the winner. :-)

**Comment by KaKi87** [comment]
Amazing ! Thanks !!

**Comment by pvi-github** [comment]
You might also try Leantime : https://github.com/Leantime/leantime

**Comment by KaKi87** [comment]
Also too much when only wanting a kanban.

**Comment by sebastian-de** [comment]
https://github.com/kanbn/kan looks very promising.

**Comment by KaKi87** [comment]
It's still a very fresh project with very few features. It didn't even have mobile support when I tried it, I see it got added later on, maybe I'll have another look.

Since then though, I've been using [RARgames/4gaBoards](https://github.com/RARgames/4gaBoards) and am very satisfied, and the maintainer is very responsive to bug reports and feature requests.

**Comment by wilsonwu** [comment]
recommend a project, github.com/letscrum/letscrum-ui

**Comment by chmac** [comment]
Thanks @sebastian-de and @KaKi87 those look worth checking out, kan especially looks cool.
